changed transmission_availability

spacemanspiff2007 opened the pull request:
Should now show the same as the gui

spacemanspiff2007 commented on flexget/plugins/clients/transmission.py:
```diff
@@ -234,8 +234,11 @@ def on_task_input(self, task, config):
                         'error when requesting transmissionrpc attribute {}', attr
                     )
+
+            bytes_completed = sum(map(lambda x: x['bytesCompleted'] if x['wanted'] else 0, t.fileStats))
```
I am unsure about the ```if x['wanted'] else 0``` part

gazpachoking commented:
Sorry about the lack of feedback on this. If you've tested, and this is consistent with the UI and working for you, I'm okay with it. Mind updating it to resolve the conflicts?

spacemanspiff2007 commented:
I've switched to a stand alone python script to do these calculations but I've updated the PR with what I am currently using.
However this is untested.
